app/ruby/app-issue-creator/server.rb

```diff
@@ -0,0 +1,113 @@
+require 'sinatra'
+require 'jwt'
+require 'json'
+require 'active_support/all'
+require 'octokit'
+
+begin
+  GITHUB_APP_ID = ENV.fetch("GITHUB_APP_ID")
+  path_to_pem = './private-key.pem'
```

Comment:
@azizshamim I've updated the script to use an environment variable and included instructions in the Readme to set this.